lib/engine/game/g_18_eus/step/special_token.rb

```diff
@@ -32,10 +39,15 @@ def ability(entity)
 
             possible_times = [
               '%current_step%',
-              'owning_corp_or_turn',
+              'current_corp_or_turn',
             ]
```

Comment:
What is the difference here?

Reply:
The old way is correct. Not sure where I got the other.

Reply:
Not needed. Removing.